src/Features/Core/Portable/PullMemberUp/PullMembersUpAnalysisResult.cs

```diff
@@ -6,9 +6,9 @@
 namespace Microsoft.CodeAnalysis.PullMemberUp
 {
     /// <summary>
-    /// This is struct contains all the operations needs to be done on members and destination to complete the pull up operation.
+    /// This class contains all the operations needs to be done on members and destination to complete the pull up operation.
     /// </summary>
-    internal readonly struct PullMembersUpAnalysisResult
+    internal class PullMembersUpAnalysisResult
```

Comment:
why this change?

Reply:
I am trying to use 'null' to express user has clicked cancel button. So I feel it is not good to use the 'default' value of a struct so I change it to class

Reply:
Needs comment to htat effect.  Thanks!

Reply:
Sure